flask_advanced/blueprint/Products/utils.py

```diff
@@ -0,0 +1,13 @@
+import json
+
+
+def get_data():
+    with open("blueprint/Products/data.json") as file1:
+        return json.load(file1)
+
+
+def add_data(data):
+    with open("blueprint/Products/data.json", 'w', encoding='utf-8') as f:
```

Comment:
when you write data to file you need to catch some types of error like `file not found`